be/src/exec/es/es_scroll_parser.cpp

```diff
@@ -233,17 +262,18 @@ int ScrollParser::get_total() {
 
 
 Status ScrollParser::fill_tuple(const TupleDescriptor* tuple_desc, 
-            Tuple* tuple, MemPool* tuple_pool, bool* line_eof) {
+            Tuple* tuple, MemPool* tuple_pool, bool* line_eof, const std::map<std::string, std::string>& docvalue_context) {
     *line_eof = true;
     if (_size <= 0 || _line_index >= _size) {
         return Status::OK();
     }
 
     const rapidjson::Value& obj = _inner_hits_node[_line_index++];
-    const rapidjson::Value& line = obj[FIELD_SOURCE];
-    if (!line.IsObject()) {
-        return Status::InternalError("Parse inner hits failed");
+    bool pure_doc_value = false;
```

Comment:
we need check this every tuple? I think it should be checked once.

Reply:
Maybe need two function part if checked once

Reply:
I would add this todo in future, currently modify this will consume mount of time because the code structure